tests/e2e/utils/helpers.js

```diff
@@ -1,187 +1,11 @@
-/**
- * External dependencies
- */
-import { pressKeyWithModifier } from '@wordpress/e2e-test-utils';
-
-/**
- * Perform a "select all" and then fill a input.
- *
- * @param {string} selector Css selector
- * @param {string} value Input value
- */
-export const clearAndFillInput = async ( selector, value ) => {
-	await page.focus( selector );
-	await pressKeyWithModifier( 'primary', 'a' );
-	await page.type( selector, value );
-};
-
-/**
- * Click a tab (on post type edit screen).
- *
- * @param {string} tabName Tab label
- */
-export const clickTab = async ( tabName ) => {
-	await expect( page ).toClick( '.wc-tabs > li > a', { text: tabName } );
-};
-
-/**
- * Save changes on a WooCommerce settings page.
- */
-export const settingsPageSaveChanges = async () => {
-	await page.focus( 'button.woocommerce-save-button' );
-	await Promise.all( [
-		page.waitForNavigation( { waitUntil: 'networkidle0' } ),
-		page.click( 'button.woocommerce-save-button' ),
-	] );
-};
-
-/**
- * Save changes on Permalink settings page.
- */
-export const permalinkSettingsPageSaveChanges = async () => {
-	await page.focus( '.wp-core-ui .button-primary' );
-	await Promise.all( [
-		page.waitForNavigation( { waitUntil: 'networkidle0' } ),
-		page.click( '.wp-core-ui .button-primary' ),
-	] );
-};
-
-/**
- * Set checkbox.
- *
- * @param {string} selector CSS selector
- */
-export const setCheckbox = async ( selector ) => {
-	await page.focus( selector );
-	const checkbox = await page.$( selector );
-	const checkboxStatus = await (
-		await checkbox.getProperty( 'checked' )
-	 ).jsonValue();
-	if ( true !== checkboxStatus ) {
-		await page.click( selector );
-	}
-};
-
-/**
- * Unset checkbox.
- *
- * @param {string} selector CSS selector
- */
-export const unsetCheckbox = async ( selector ) => {
-	await page.focus( selector );
-	const checkbox = await page.$( selector );
-	const checkboxStatus = await (
-		await checkbox.getProperty( 'checked' )
-	 ).jsonValue();
-	if ( true === checkboxStatus ) {
-		await page.click( selector );
-	}
-};
-
-/**
- * Wait for UI blocking to end.
- */
-export const uiUnblocked = async () => {
-	await page.waitForFunction(
-		() => ! Boolean( document.querySelector( '.blockUI' ) )
-	);
-};
-
 /**
  * Wait for UI placeholders to finish and UI content is loaded.
+ *
+ * Eventually this will be included in the @woocommerce/e2e-utils package and will be moved there.
+ * See: https://github.com/woocommerce/woocommerce/issues/29751
  */
-export const uiLoaded = async () => {
+export const uiWCAdminLoaded = async () => {
```

Comment:
AFAIK the class referenced here is specific to WCPay [[code](https://github.com/Automattic/woocommerce-payments/blob/bff6eb949af53728237ad44073b52b87b40cd714/client/components/loadable/index.js#L22-L24)], and so this function isn't directly transferable to the generic package. Noted in https://github.com/woocommerce/woocommerce/issues/29751#issuecomment-830229063 as well

Reply:
Sounds good. I can update the comment and I think we can leave functions specific to WCPay within `helpers.js`, if that works.

Reply:
I made these changes, updating the comment and method name to`elementUnblocked()`.

Reply:
`uiLoaded` was descriptive of the behavior being awaited, while `elementUnblocked` seems like it should accept a selector (like the one proposed for the core package in https://github.com/woocommerce/woocommerce/pull/29725 does) – can the name stay, or does it sound too ambiguous?

Reply:
I think the name works, I've updated it to `uiLoaded()`.